Governance/manage/manage-intro.md

```diff
@@ -0,0 +1,30 @@
+# Guidelines to manage climate dataset
+
+These guidelines are aimed to dataset managers and cover all aspects of managing an authoritative or a replicated dataset on a local server.
+Some of examples we show here are NCI based as this is where we managed most of our data. Hopefully the examples can still be extrapolated to other platforms. 
+data custodians and stewards in technical data management at NCI:
+
+Project handling 
+  writers group creation
+  use of posix permissions
+  use of ACLs
+  caveats with ACLs - what breaks them and when do they break?
+Note, this is NCI specific content - on CSIRO internal resources e.g., access mgmt is governed by windows ACLs - email schelp for advice.
```

Comment:
"email [SChelp](mailto:schelp@csiro.au)"

Reply:
I fixed the typo and put this in a separate box, I'm not sure what the best placement for this is to be true